x-pack/plugins/cases/public/components/connectors/resilient/case_fields.tsx

```diff
@@ -77,12 +77,15 @@ const ResilientFieldsComponent: React.FunctionComponent<ConnectorFieldsProps> =
             field.setValue(changedOptions.map((option) => option.value as string));
           };
 
-          const selectedOptions = (field.value ?? []).map((incidentType) => ({
-            value: incidentType,
-            label:
-              (allIncidentTypes ?? []).find((type) => incidentType === type.id.toString())?.name ??
-              '',
-          }));
+          const selectedOptions =
```

Comment:
Made this change to fix weird behaviour in UI:
![image](https://github.com/elastic/kibana/assets/117571355/0aaf4f22-9cf6-4e70-b4f8-bfd5fde83871)